examples/ota/update/src/main.rs

```diff
@@ -8,15 +8,16 @@
 //! Adjust the target and the chip in the following commands according to the
 //! chip used!
 //!
-//! - `cargo xtask build examples examples esp32 --example=gpio_interrupt`
-//! - `espflash save-image --chip=esp32 examples/target/xtensa-esp32-none-elf/release/gpio_interrupt
-//!   examples/target/ota_image`
-//! - `cargo xtask build examples examples esp32 --example=ota_update`
-//! - `espflash save-image --chip=esp32 examples/target/xtensa-esp32-none-elf/release/ota_update
+//! - `cargo xtask build examples gpio --chip=esp32`
+//! - `espflash save-image --chip=esp32
+//!   examples/interrupt/gpio/target/xtensa-esp32-none-elf/release/gpio_interrupt
```

Comment:
Hm, it would be better to not split this command imo 🤔

Reply:
yes - but rustfmt thinks it's a good idea